app/src/main/java/com/example/giphyservice/ui/list/GifsAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.giphyservice.ui.list
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.giphyservice.data.model.Gif
+import com.example.giphyservice.R
+
+// Adapter - оброблює дані і зв'язує зі списком
+//
+class GifsAdapter(
+    var mListener: OnItemClickListener,
+    val context: Context,
+) : RecyclerView.Adapter<GifViewHolder>() {
+    val gifs: MutableList<Gif> = mutableListOf()
+
+    fun updateGifs(gif: List<Gif>) {
```

Comment:
please rearrange members of that class to have:
- fields
- overriden methods
- public methods
- private methods
- companion object